x/ccv/provider/keeper/proposal_test.go

```diff
@@ -348,39 +334,23 @@ func TestHandleConsumerRemovalProposal(t *testing.T) {
 				hourFromNow,
 			).(*providertypes.ConsumerRemovalProposal),
 			blockTime: now, // Before proposal's stop time
-			expStop:   false,
 		},
 	}
 
 	for _, tc := range tests {
 
 		// Common setup
 		keeperParams := testkeeper.NewInMemKeeperParams(t)
-		providerKeeper, ctx, ctrl, mocks := testkeeper.GetProviderKeeperAndCtx(t, keeperParams)
+		providerKeeper, ctx, ctrl, _ := testkeeper.GetProviderKeeperAndCtx(t, keeperParams)
 		providerKeeper.SetParams(ctx, providertypes.DefaultParams())
 		ctx = ctx.WithBlockTime(tc.blockTime)
 
-		// Mock expectations and setup for stopping the consumer chain, if applicable
-		if tc.expStop {
-			testkeeper.SetupForStoppingConsumerChain(t, ctx, &providerKeeper, mocks)
-		}
-		// Note: when expStop is false, no mocks are setup,
-		// meaning no external keeper methods are allowed to be called.
-
 		err := providerKeeper.HandleConsumerRemovalProposal(ctx, tc.prop)
 		require.NoError(t, err)
 
-		if tc.expStop {
-			// Expect no pending proposal to exist
-			found := providerKeeper.GetPendingConsumerRemovalProp(ctx, tc.prop.ChainId, tc.prop.StopTime)
-			require.False(t, found)
-
-			testProviderStateIsCleaned(t, ctx, providerKeeper, tc.prop.ChainId, "channelID")
```

Comment:
In replacing these lines we want to maintain the same amount of coverage. Here the proposal actually was processed so `testProviderStateIsCleaned` checks that the consumer was removed. An equivalent should be added. ie.

```
BeginBlocker()
testProviderStateIsCleaned
```

to check the same thing.